### ISSUE TITLE
Editor effect not same in Android Devices!

### ISSUE THREAD

**Issue by rigby00** [keijiro/KinoBloom#17]
Hello, first thanks for this awesome Bloom effect. In mobile we got +30 - +40 FPS. 

Problem is i'm setting bloom effect on editor but in Android Devices it's looking more light. How to solve this problem.

Thanks.

**Comment by keijiro** [comment]
FYI: This project hasn't been maintained for several years. I'd recommend using the post processing stack instead. There is no advantage in using KinoBloom compared to the bloom effect in the post processing stack.

**Comment by rigby00** [comment]
> FYI: This project hasn't been maintained for several years. I'd recommend using the post processing stack instead. There is no advantage in using KinoBloom compared to the bloom effect in the post processing stack.

Unity's Standart Bloom effect is killing FPS on android devices, i'm getting 15-25 FPS. But with this KinoBloom i'm getting 60-70 FPS. So i think it's best to use for android devices but i have this problem. "Editor effect not same in Android Devices"

Thanks.

**Comment by keijiro** [comment]
That's weird. The bloom effect in the post processing stack was improved/optimized compared to KinoBloom.

**Comment by rigby00** [comment]
> That's weird. The bloom effect in the post processing stack was improved/optimized compared to KinoBloom.

Yes it's. Can you fix this gamma issue? I mean it's much more lightly on android devices.

**Comment by rigby00** [comment]
-DoublePost
@keijiro I really need your help i tested every bloom effect someone just give black screen. Your bloom effect are amazing and working on mobile too but this gamma issue is killing me. I will be happy if you can fix this. Thanks.

**Comment by ljdong-fy** [comment]
black screen on android devices
